src/Pusher.php

```diff
@@ -140,11 +142,20 @@ public function getSettings()
 
     /**
      * Set a logger to be informed of internal log messages.
+     * The logger must have a log method.
+     *
+     * @param object $logger A object with a public function log($message) method
+     *
+     * @throws PusherException Throws exception if logger object does not have a log method, this exception can be
+     *                         safely ignores if the the passed object always have a log method
```

Comment:
ignores -> ignored

Reply:
In fact I don't think the description after the `,` is necessary. Unless I'm missing something?

Reply:
I think the description after the "," is important, people could either wrap it in a try catch block to get rid of the IDE warning, or they could tell the ide to suppress the warning

Reply:
Ok, fair enough. Could you fix the typo then?